authorization-server/server.js

```diff
@@ -8,16 +8,30 @@ import jwtAuthorizationGrant from './jwt-authorization-grant.js';
 import routes from './routes.js';
 import makeConfiguration from './server-configuration.js';
 import tokenExchange from './token-exchange.js';
+import {createClient} from "redis"
 
 const __dirname = dirname(import.meta.url);
 
 
 const PORT = process.env.AUTH_SERVER_PORT;
 const ISSUER = process.env.AUTH_SERVER;
+const cookieSecret = process.env.COOKIE_SECRET;
 
 const app = express();
 
-const cookieSecret = process.env.COOKIE_SECRET;
+let redisClient;
+
+(async () => {
+  redisClient = createClient();
```

Comment:
does the createClient call need to be in this anonymous function?  Seems like it can be directly assigned in line 22 above